radix-transaction-scenarios/assets/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+
+cd "$(dirname "$0")"
+
+IMAGE_NAME="radixdlt/scrypto-builder"
+IMAGE_TAG="v1.1.1"
+WORKSPACE_DIR="blueprints"
+DESTINATION_DIR="."
+for PACKAGE_NAME in "flash_loan" "global_n_owned" "kv_store" "max_transaction" "metadata" "radiswap" "royalties"
```

Comment:
I think it would be simpler and a little more accurate if this was a `build_package.sh` script, for use in development... and as part of running it you have to provide a package name and an image tag, and then the script outputs the `X.rpd` and `X.wasm`.

If we care about verification, the `build_package.sh` script could also output an artifact such as `X.build_info.yaml` which explains which version of the scrypto builder was used to build the file. Separately, we could have a `verify_package_builds.sh` script which greps for `X.wasm` files, loads the `X.build_info.yaml`, runs the build, with the right version of scrypto builder, and checks for equality. (But to be honest, I don't think this is needed for transaction scenarios so much).

Of course, we hit the problem of the existing files not having reproducible builds - in which case, I guess we could have a `build_info.yaml` which instructs us not to verify the build or something.

Anyway, as mentioned, I don't think it's critical to support verification on these builds for now -- but I think possibly it's worth us changing this script to just build a single package using a specific builder version?

Reply:
Will update to the script to take an argument for a single package.